src/lib/math.js

```diff
@@ -18,8 +18,41 @@ const $builtinmodule = function (name) {
         return new Sk.builtin.float_(Math.ceil(_x));
     };
 
-    function comb(x, y) {
-        throw new Sk.builtin.NotImplementedError("math.comb() is not yet implemented in Skulpt");
+    function comb(n, k) {
+        n_js = Sk.misceval.asIndexOrThrow(n);
+        k_js = Sk.misceval.asIndexOrThrow(k);
+        if (n_js < 0) {
+            throw new Sk.builtin.ValueError("n must be an non-negative integer");
+        }
+        if (k_js < 0) {
+            throw new Sk.builtin.ValueError("k must be a non-negative integer");
+        }
+        if (k_js > n) {
+            return new Sk.builtin.int_(0);
+        }
+        n = new Sk.builtin.int_(n_js);
+        k = new Sk.builtin.int_(k_js);
+        let temp = n.nb$subtract(k);
+        if (Sk.ffi.remapToJs(temp) < k_js) {
+            k = temp;
+            k_js = Sk.ffi.remapToJs(k);
+        }
```

Comment:
consider `let temp = Sk.ffi.remapToJs(n.nb$subtract(k));`
We don't use the `k` python object after this line.

Reply:
Yeah, that's much nicer :)